caikit/runtime/names.py

```diff
@@ -215,7 +221,7 @@ def get_task_predict_request_name(
 ##  Service Definitions
 
 TRAINING_MANAGEMENT_SERVICE_NAME = "TrainingManagement"
-TRAINING_MANAGEMENT_PACKAGE = "caikit.runtime.training"
+TRAINING_MANAGEMENT_SERVICE_PACKAGE = "caikit.runtime.training"
```

Comment:
nit: Is it worth renaming these to `*_PACKAGE_NAME` to make it clear that they're string constants?

Reply:
We've used just `package` in this context in a lot of places (e.g. `package.py`). I can see that being confusing for those not in the guts, but I don't really want to go find all the places this is under-specified in this PR. Ok to skip for now?

Reply:
sure 👍